app/controllers/application_controller.rb

```diff
@@ -1706,8 +1706,8 @@ def get_view(db, options = {})
       adv_search_build(db)
     end
     if @edit && !@edit[:selected] && # Load default search if search @edit hash exists
-       @settings.fetch_path(:default_search, db.to_sym) # and item in listnav not selected
-      load_default_search(@settings[:default_search][db.to_sym])
+       settings(:default_search, db.to_sym) # and item in listnav not selected
```

Comment:
was not changing that line and other tests around are aligned the same way

I'd rather not change that in this PR